packages/hvsock/hvsock.3.0.1/opam

```diff
@@ -0,0 +1,64 @@
+opam-version: "2.0"
+maintainer: "dave@recoil.org"
+authors: [ "David Scott" "Rolf Neugebauer" "Anil Madhavapeddy" "Simon Ferquel"]
+license: "ISC"
+homepage: "https://github.com/mirage/ocaml-hvsock"
+dev-repo: "git+https://github.com/mirage/ocaml-hvsock.git"
+bug-reports: "https://github.com/mirage/ocaml-hvsock/issues"
+doc: "https://mirage.github.io/ocaml-hvsock"
+
+build: [
+  [ "dune" "subst" ] {dev}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "base-bytes"
+  "base-threads"
+  "base-unix"
+  "lwt" {>= "3.2.0"}
+  "logs"
+  "fmt"
+  "cmdliner" {>= "1.1"}
+  "sha"
+  "uri"
+  "base64" {>= "3.0.0"}
+  "uuidm"
+  "uutf"
+  "mirage-flow" {>= "2.0.0"}
+  "mirage-flow-combinators" {>= "2.0.0"}
+  "mirage-time" {>= "2.0.0"}
+  "cstruct" {>= "6.0.0"}
+  "duration"
+  "dune" {>= "1.2.0"}
+  "alcotest" {with-test & >= "0.4.0"}
+]
+depexts: [
```

Comment:
Why not depending on [`conf-linux-libc-dev`](https://github.com/ocaml/opam-repository/blob/master/packages/conf-linux-libc-dev/conf-linux-libc-dev.0/opam) instead?

Reply:
I've not seen that before :) Thanks, I'll give it a go!